platform/darwin/resources/lt.lproj/Foundation.stringsdict

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>COORD_DEG_LONG</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@degrees@</string>
+		<key>degrees</key>
+		<dict>
```

Comment:
Sure. MGLCoordinateFormatter uses this string to produce, for example, “54 degrees and 41 minutes north by 25 degrees and 19 minutes east”. (Normally, you’d use the short format “54°41′N 25°19′E” in UI, but the SDK’s built-in VoiceOver support needs to spell out the message to ensure correct voice output.)

Unfortunately, NSFormatter subclasses don’t typically provide a way to configure the case, so people usually translate into something generic. For its part, the VoiceOver support says things like “You are here: 54 degrees…” or “[annotation title]: 54 degrees…”. Possessive case would seem like an odd choice to me, but I’m unfamiliar with Lithuanian grammar and assumed (perhaps wrongly) that CLDR’s contributors had chosen these declensions deliberately.